scripts/clickhouse_operator_sync.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+#
+# This tool fetch and format 'Altinity/clickhouse-operator'
+# k8s resource definitions into our chart.
+#
+# Why do we need this? The 'clickhouse-operator' doesn't expose a Helm
+# package so we need to collect and bundle the resources by our own.
+#
+
+set -o errexit
+set -o nounset
+
+CURRENT_DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
```

Comment:
non-blocking

Use `pwd -P` here or remove from the other, given it's the default?